app/h_terminal.py

```diff
@@ -14,8 +16,8 @@ async def run(socket, path, services):
         cmd = await socket.recv()
         handler = services.get('term_svc').socket_conn.tcp_handler
         paw = next(i.paw for i in handler.sessions if i.id == int(session_id))
-        services.get('contact_svc').report['websocket'].append(
-            dict(paw=paw, date=datetime.now().strftime('%Y-%m-%d %H:%M:%S'), cmd=cmd)
+        services.get('contact_svc').report['WEBSOCKET'].append(
```

Comment:
I had to make this capitalized because when fetching contact reports, the contact names are capitalized for searches.